src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Included `isRemote` check in `PropagateOrIgnoreData` in `TracerProviderSDK` to
```

Comment:
TracerProvider modified to create activity irrespective of SamplingResult, for spans with Remote Parent, to maintain context propagation.

^ or similar wording.